### PR TITLE
Nightly - Offline - Attempt to fix CI UnknownHostException.

### PR DESCRIPTION
**Pull request by kdeakinstructure**
java.net.UnknownHostException: Unable to resolve host "http-inputs-inst.splunkcloud.com": No address associated with hostname

Probably the issue was that the device was offline when the test has failed so it cannot open the connection for the Splunk URL.

### CODE REVIEW

**Comment by kdeakinstructure** [comment]
I have run the all the E2E tests and there weren't any fails which has been caused by this change.

File: automation/espresso/src/main/kotlin/com/instructure/canvas/espresso/CanvasTest.kt [hunk 1]
```diff
@@ -464,6 +498,12 @@ abstract class CanvasTest : InstructureTestingContract {
 
         private var configChecked = false
 
+        val ENABLE_WIFI_COMMAND: String = "svc wifi enable"
```
Comment (kristofnemere) [inline automation/espresso/src/main/kotlin/com/instructure/canvas/espresso/CanvasTest.kt:501]: these could be private const

**Review by kristofnemere — APPROVED** [review]

**Review by tamaskozmer — APPROVED** [review]

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Pull request merged**